web/src/user_profile.ts

```diff
@@ -546,6 +551,8 @@ export function show_user_profile(user: User, default_tab_key = "profile-tab"):
         (people.can_admin_user(user) || settings_data.user_can_subscribe_other_users()) &&
         !user.is_system_bot &&
         people.is_person_active(user.user_id);
+    show_user_group_container =
+        invite_user_group_picker_pill.get_user_groups_allowed_to_add_members().length > 0;
```

Comment:
Might be good to move `get_user_groups_allowed_to_add_members()` function to `user_groups` in a prep commit.

Reply:
We have all the data functions like `get_user_groups_allowed_to_mention`, `get_potential_subgroups` etc. in `user_groups.ts`, so that might be a better place.